src/Service/SessionUserProvider.php

```diff
@@ -59,12 +56,26 @@ public function loadUserByUsername($userId)
             return new SessionUser($user, $this->userRepository);
         }
 
-        throw new UsernameNotFoundException(
+        throw new UserNotFoundException(
             sprintf('Username "%s" does not exist.', $userId)
         );
     }
 
-    public function refreshUser(UserInterface $user)
+    public function loadUserByIdentifier($identifier): SessionUserInterface
+    {
+        /** @var IliosUser $user */
```

Comment:
duplicates functionality if `loadUserByUsername()`, suggesting one of them becoming a wrapper for the other.

Reply:
Thanks! Fixed.